include/oboe/StabilizedCallback.h

```diff
@@ -22,31 +22,25 @@
 
 namespace oboe {
 
-class StabilizedCallback : public AudioStreamCallback {
+class StabilizedCallback : public AudioStreamDataCallback {
```

Comment:
This is an API change that will cause source level  breaks.
Maybe add a StabilizedDataCallback and deprecate the older compatible definition that uses AudioStreamCallback..

Reply:
Good spot, thanks.